.travis.yml

```diff
@@ -2,8 +2,9 @@ language: ruby
 rvm:
   - "2.1"
   - "2.2"
+  - "2.5.1"
```

Comment:
how about to add `2.4.4` as well?

Reply:
Didn't cause problems so far but sure, why not